### PR TITLE
HW Submission from Dani Kim

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
+body {
+	background-color: #efefef;
+	margin: 0px;
+}
+h1 {
+	text-align: center;
+	font-family: 'Raleway', sans-serif;
```
Comment (qodesmith) [inline css/style.css]: @daniwhkim Nice usage of google fonts :)

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	font-size: 68pt;
+	margin: 20px;
+	color: #663399;
+}
+nav {
+	text-align: center;
+	background-color: grey;
```
Comment (qodesmith) [inline css/style.css:14]: @daniwhkim grey or gray? Let the debate begin...

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+	padding: 10px;
+}
+a {
+	text-decoration: none;
+	color: black;
+	margin-right: 25px;
+	margin-left: 25px;
+	font-family: 'Raleway', sans-serif;
```
Comment (qodesmith) [inline css/style.css]: :hammer: It looks like you want the Raleway font for the whole page. If so, then just put it on the `body`.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+	margin-top: 10px;
+}
+
+/*Filmography*/
+img.portfolio {
+	width: 70%;
+	margin-top: 10px;
+}
+ul {
+	list-style-type: none;
```
Comment (qodesmith) [inline css/style.css:67]: @daniwhkim Nice. Bye bye bullets.

File: filmography.html [hunk 5]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>
```
Comment (qodesmith) [inline filmography.html]: @daniwhkim For content this small, you don't have to use separate lines with indentation. You can just do it on a single line like this:

``` html
<title>Filmography</title>
```

File: index.html [hunk 6]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```
Comment (qodesmith) [inline index.html:1]: @daniwhkim Excellent indentation in your html files. Your page looked awesome. Nice & clean with those responsive images! Love the grid with the movies. Great usage of making the `<li>`'s inline-blocks and all of the m@ 30%. Really creative thinking there.

File: css/style.css (hunk 3 above)
Comment (ghost) [inline css/style.css]: @qodesmith Done!

File: index.html (hunk 6 above)
Comment (ghost) [inline index.html:1]: @qodesmith Thank you Aaron! I appreciate your feedback! See you on Tuesday.

File: filmography.html (hunk 5 above)
Comment (ghost) [inline filmography.html]: @qodesmith Also done!

**Comment by ghost** [comment]
@qodesmith Hi Aaron, have all my changes gone through? Approved?

**Comment by qodesmith** [comment]
Hey,

I just approved it. Sorry for the delay!

Aaron

**Pull request closed without merging**